mllib/src/main/scala/org/apache/spark/ml/regression/GeneralizedLinearRegression.scala

```diff
@@ -620,25 +779,67 @@ object GeneralizedLinearRegression extends DefaultParamsReadable[GeneralizedLine
   private[regression] object Link {
 
     /**
-     * Gets the [[Link]] object from its name.
+     * Gets the [[Link]] object based on link or linkPower.
```

Comment:
```
Gets the Link object based on param family, link and linkPower.
If param family was set with "tweedie", return or construct link function object according to linkPower; otherwise, return link function object according to link.
```